docker-compose.rest-api.yaml

```diff
@@ -0,0 +1,34 @@
+version: '3.0'
+
+services:
+    nginx:
+        image: nginx:stable-alpine
+        container_name: sdk-nginx
+        ports:
+            - '8080:80'
+        volumes:
+            - ./public:/data/public
+            - ./infrastructure/web/nginx/default.conf:/etc/nginx/conf.d/default.conf
+        depends_on:
+            - php-fpm
+        restart: always
+
+    php-fpm:
+        container_name: sdk-php-fpm
+        build:
+            context: .
+            dockerfile: infrastructure/web/php-fpm.Dockerfile
+        volumes:
+            - ./:/data
```

Comment:
Must be specific per environment. `./:/data` binding ok for development, but for the prod mode, it shouldn't be like that, because the prod mode should use prebuild image instead and bind only specific files.

Please rename this file to `docker-compose.rest-api.dev.yaml` and create the separate task

Reply:
Fixed